Gruntfile.js

```diff
@@ -85,8 +85,83 @@ module.exports = function(grunt) {
             }
         },
         clean: {
+            coverage: ['coverage/', 'coverage.lcov', '.nyc_output/'],
             dist: ['.dist/'],
+            packages: ['Consus-Client-*/'],
             test: ['.test/']
+        },
+        electron: {
+            win64Build: {
+                options: {
+                    name: 'Consus-Client',
+                    dir: '.',
+                    out: '.',
+                    version: '1.4.2',
+                    platform: 'win32',
+                    arch: 'x64',
+                    prune: true,
+                    overwrite: true,
+                    asar: true,
+                    icon: 'assets/icons/consus-logo'
```

Comment:
If only we had turned `branding` into an npm module and reused it here... Oh well.